pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/TlsIntegrationTest.java

```diff
@@ -211,7 +211,7 @@ protected PinotConfiguration getDefaultServerConfiguration() {
         CertBasedTlsChannelAccessControlFactory.class.getName());
     prop.put("pinot.server.adminapi.access.protocols", "internal");
     prop.put("pinot.server.adminapi.access.protocols.internal.protocol", "https");
-    prop.put("pinot.server.adminapi.access.protocols.internal.port", "7443");
+    prop.put("pinot.server.adminapi.access.protocols.internal.port", "9443");
```

Comment:
I always get the issue in binding to this port on my local Mac machine with Intellij, Is it okay to change the port to 9443?

Reply:
I plan to randomize ports for tests: https://github.com/apache/pinot/pull/11861

Reply:
Reversed my change. Thanks